oneview-sdk-java-lib/src/main/java/com/hp/ov/sdk/util/SdkUtils.java

```diff
@@ -117,11 +116,7 @@ private void validateApiVersion(final int actualVersion, final int expectedVersi
             logger.info("API version : " + "You are trying to connect to appliance verion: " + actualVersion
```

Comment:
Thanks for pointing the minimum version =)

The SDK only supports API 1.2 for now.
If the user decides to use another API version, the result will probably be a parsing (JSON / Java) error or a message regarding a missing field from 2.0.

We will work on a solution for that.

Again, thanks!


From: dronin [mailto:notifications@github.com]
Sent: segunda-feira, 25 de janeiro de 2016 12:40
To: HewlettPackard/oneview-sdk-java
Cc: Kovalski, Filipe
Subject: Re: [oneview-sdk-java] Removed API version validation. (#3)


In oneview-sdk-java-lib/src/main/java/com/hp/ov/sdk/util/SdkUtils.java<https://github.com/HewlettPackard/oneview-sdk-java/pull/3#discussion_r50700410>:

> @@ -117,11 +116,7 @@ private void validateApiVersion(final int actualVersion, final int expectedVersi

>              logger.info("API version : " + "You are trying to connect to appliance verion: " + actualVersion
The correct test would be : applianceDTO.getMinimumVersion() <= requestedVersion <= applianceDTO.getCurrentVersion() But we also need to make sure the requested version is supported by the library not just the appliance. How do we handle a request for a API version that returns an object version not supported by the library? From: Luiz Hermes Svoboda Junior [mailto:notifications@github.com] Sent: Monday, January 25, 2016 7:10 AM To: HewlettPackard/oneview-sdk-java Subject: Re: [oneview-sdk-java] Removed API version validation. (#3<https://github.com/HewlettPackard/oneview-sdk-java/pull/3>) In oneview-sdk-java-lib/src/main/java/com/hp/ov/sdk/util/SdkUtils.java<#3 (comment)<https://github.com/HewlettPackard/oneview-sdk-java/pull/3#discussion_r50686063>>:
@@ -117,11 +116,7 @@ private void validateApiVersion(final int actualVersion, final int expectedVersi logger.info("API version : " + "You are trying to connect to appliance verion: " + actualVersion
We could change the versions equality test to a less than or equal. Doing that we would still have a log entry message when the actual version is less than the expected version (which is a valid usage of the SDK). — Reply to this email directly or view it on GitHub<https://github.com/HewlettPackard/oneview-sdk-java/pull/3/files#r50686063>.

—
Reply to this email directly or view it on GitHub<https://github.com/HewlettPackard/oneview-sdk-java/pull/3/files#r50700410>.